pkg/util/validation.go

```diff
@@ -18,19 +18,32 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const LabelValueFmt string = "[-A-Za-z0-9_.]*"
```

Comment:
The more I think on this, the more I dislike the idea of label values of "." being valid.  Can we define this the same way as qualifiedNamePiece?

Maybe define LabelValueFmt fully and then define qualifiedNamePiece = LabelValueFmt